zwave_js_server/model/controller.py

```diff
@@ -128,6 +128,130 @@ def supports_timers(self) -> Optional[bool]:
         """Return supports_timers."""
         return self.data.get("supportsTimers")
 
+    async def async_begin_inclusion(
+        self, include_non_secure: Optional[bool] = None
+    ) -> bool:
+        """Send beginInclusion command to Controller."""
+        data = await self.client.async_send_command(
+            {
+                "command": "controller.begin_inclusion",
+                "includeNonSecure": include_non_secure,
+            }
+        )
+        return bool(data["success"])
+
+    async def async_stop_inclusion(self) -> bool:
+        """Send stopInclusion command to Controller."""
+        data = await self.client.async_send_command(
+            {"command": "controller.stop_inclusion"}
+        )
+        return bool(data["success"])
+
+    async def async_begin_exclusion(self) -> bool:
+        """Send beginExclusion command to Controller."""
+        data = await self.client.async_send_command(
+            {"command": "controller.begin_exclusion"}
+        )
+        return bool(data["success"])
+
+    async def async_stop_exclusion(self) -> bool:
+        """Send stopExclusion command to Controller."""
+        data = await self.client.async_send_command(
+            {"command": "controller.stop_exclusion"}
+        )
+        return bool(data["success"])
+
+    async def async_remove_failed_node(self, node: Union[Node, int]) -> None:
```

Comment:
Why would we accept both? Does Z-Wave JS accepts both too ?

Reply:
You are right, I've just copied it from node's `async_set_value`. Will remove it then ;)